### PR TITLE
feat(agoric-cli): add `ibc-setup` and `ibc-relayer` commands with docs

### PR DESCRIPTION
**Pull request by michaelfig**
## Description

Introduces a little workflow that developers can go through to test IBC from their workstation to a public chain.

### Security Considerations

n/a

### Scaling Considerations

n/a

### Documentation Considerations

The `agoric ibc-relayer` and `agoric ibc-setup` subcommands should be documented, with some source material from `agoric-sdk/packages/agoric-cli/README.md`.

### Testing Considerations

Manually tested between my workstation and ollinet.

### CODE REVIEW

**Review by arirubinstein — APPROVED** [review]

**Comment by dckc** [comment]
The documentation should make it to https://docs.agoric.com/guides/agoric-cli/ in due course, right? The `agoric` CLI is an externally visible API, so adding this is a commitment to a stable interface, right?

File: packages/agoric-cli/registry.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+version: 1
+
+chains:
+  ollinet:
```
Comment (dckc) [inline packages/agoric-cli/registry.yaml:4]: should 'ollinet' become part of our external API?

File: packages/agoric-cli/package.json [hunk 2]
```diff
@@ -35,10 +35,10 @@
     "@agoric/cosmic-proto": "^0.2.1",
     "@agoric/ertp": "^0.15.3",
     "@agoric/internal": "^0.2.1",
-    "@endo/nat": "^4.1.0",
     "@agoric/smart-wallet": "^0.4.2",
     "@agoric/swingset-vat": "^0.30.2",
     "@agoric/zoe": "^0.25.3",
+    "@confio/relayer": "^0.7.0",
```
Comment (dckc) [inline packages/agoric-cli/package.json:41]: We're going to add a whole IBC relayer to our dependencies?

Should we talk this over a bit? Should we have an issue?

File: packages/agoric-cli/registry.yaml (hunk 1 above)
Comment (michaelfig) [reply packages/agoric-cli/registry.yaml:4]: This `registry.yaml` is just a sample, but it needs to be named `registry.yaml` or else `agoric ibc-setup` cannot use it.

File: packages/agoric-cli/package.json (hunk 2 above)
Comment (michaelfig) [reply packages/agoric-cli/package.json:41]: This relayer is a thin veneer on top of cosmjs.  If it's dragging in too much, we can probably dedupe some cosmjs versions.

**Comment by dckc** [comment]
oh yeah... this helped with https://github.com/Agoric/wallet-app/pull/63

**Pull request merged**